.github/workflows/checks.yml

```diff
@@ -96,54 +96,24 @@ jobs:
         run: |
           poetry install
 
-      # ====================== CONTRACTS v2 ====================== #
-
-      - name: Cache contracts v2
-        id: cache-contracts-v2
-        uses: actions/cache@v3
+      - uses: software-mansion/setup-scarb@v1.3.2
         with:
-          path: starknet_py/tests/e2e/mock/contracts_compiled_v2
-          key: ${{ runner.os }}-contracts-${{ hashFiles('starknet_py/tests/e2e/mock/contracts_v2') }}-${{ env.CAIRO_V2_SHA }}
```

Comment:
`CAIRO_V2_SHA` (and other envs added in previous PR in stack) is no longer needed, let's remove them